p/themes/base-theme/template.css

```diff
@@ -572,6 +572,12 @@ a.btn {
 	width: 100px;
 }
 
+.header > .item.search input {
```

Comment:
Maybe we should go for a faster & better selector. Could you try that one:

```suggestion
search[type="search"] {
```

Reply:
Yes, works as expected in Safari and would cover any other search inputs.